webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -769,22 +798,25 @@ private static void addFmcScrambleCutoutSheet(PdfWriter docWriter, Document doc,
      * @return the calculated font size that makes the text fit
      */
     private static final float FITTEXT_FONTSIZE_PRECISION = 0.1f;
-    public static float fitText(Font font, String text, Rectangle rect, float maxFontSize, boolean newlinesAllowed) {
+    private static float fitText(Font font, String text, Rectangle rect, float maxFontSize, boolean newlinesAllowed, boolean lineSpacing) {
         float minFontSize = 1f;
         float potentialFontSize;
         while(true) {
             potentialFontSize = (maxFontSize + minFontSize) / 2.0f;
             font.setSize(potentialFontSize);
 
-            LinkedList<Chunk> lineChunks = splitScrambleToLineChunks(text, font, rect.getWidth());
+            LinkedList<Chunk> lineChunks = splitTextToLineChunks(text, font, rect.getWidth());
             if(!newlinesAllowed && lineChunks.size() > 1) {
                 // If newlines are not allowed, and we had to split the text into more than
                 // one line, then potentialFontSize is too large.
                 maxFontSize = potentialFontSize;
             } else {
                 // The font size seems to be a pretty good estimate for how
                 // much vertical space a row actually takes up.
-                float totalHeight = lineChunks.size() * potentialFontSize;
+                
+                // 1.5 due line spacing
+                // hack to fit rules in fmc and not mess with anything else (particularly big cubes)
+                float totalHeight = (lineSpacing ? 1.5f : 1f) * lineChunks.size() * potentialFontSize;
```

Comment:
As discussed with you over IM, I think we should ask the PDF for it's leading here, rather than hard coding 1.5. We should also check if the input text has newlines, rather than requiring people calling this function to pass in a lineSpacing boolean.